### ISSUE TITLE
KCD Los Angeles 2023 (March 9 - 10)

### ISSUE THREAD

**Issue by jberkus** [cncf/kubernetes-community-days#453]
# KCD Organizer Checklist
Please fill in information, organizers, and sponsors as you assemble them, and then make a new comment when this application is ready for review.

### Event Details
* [x] Event Location: Pasadena, CA
* [x] Event Date and Time: March 9-10, 2023, 9am to 6pm
* [x] Expected Number of Attendees: 100
* [x] Event cost per ticket, and if you will have early bird pricing or not: $75EB, $95reg (admission is included in SCALE ticket), online attendance free
#### Event type:
* [ ] Virtual
* [ ] Hybrid
* [x] In person, with live streaming
#### Organizers Details (identify who will be the CFP admin, as well as the finanical officer)
* [x] Organizer 1: Josh Berkus, Red Hat, Kubernetes Community Manager, jberkus@redhat.com, @jberkus
* [x] Organizer 2: Guinevere Saenger, Plumi, Software Engineer, @gsaenger
* [x] Organizer 3: Steve Wong, VMWare, Software Engineer, @steve-wong
* [x] Organizer 4: Justin Garrison, AWS, DevRel, @jgarr
* [x] Organizer 5: Savitha Raghunathan, Red Hat, Software Engineer, @sraghunathan

### Terms

* [x] No other Kubernetes Community Day event in same major city (check with past organizers)
* [x] Minimum of 3 organizers from 3 different organizations
* [ ] Every organizer has an account on [Cloud Native Community Groups](https://community.cncf.io/)
* [x] Each organizer lives in or just outside of the KCD location
* [x] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer
* [x] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [x] All inquiries / communication for CNCF staff to be filed by Github for easy tracking
* [ ] All organizers have taken [Inclusive Open Source Community Orientation (LFC102)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* [ ] ~~Agree to offer (at minimum) 3 complimentary tickets to increase diversity for the event~~
* [x] Agree to ensure the final program schedule is diverse (e.g., not all speakers of one gender, one culture, or from 1 or 2 of the same company). [Here are some tips on how to create a diverse lineup.](https://docs.google.com/presentation/d/1fzT_BdavVKh3mnxxU-PBWyJq9JUfasKwHqekkbYVbw8/edit#slide=id.g56245ab439_0_106)
* [x] Agree to submit your final lineup to kcd@cncf.io for review, prior to publishing it
* [x] Agree to use Cloud Native Community Groups for ~~registration~~, scheduling, and event website
* [ ] ~~Agree to use SM Apply for CFP and to close it at least 6 weeks before the event~~

### Getting started checklist for organizers

* [x] Schedule monthly check-ins with KCD Core Organizer(s) (kcd@cncf.io)
* [ ] ~~Minimum of 3 sponsors committed (not required for virtual only events)~~
* [ ] ~~Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses~~
* [x] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel
* [ ] Create a Twitter/Mastodon handle specific to your KCD event (except for countries like China and Korea where Twitter is not used)
* [ ] Provided CNCF access to all event social handles

### Getting started checklist for CNCF:
* [x] Organizer email alias has been created: [organizers-los-angeles@kubernetescommunitydays.org](mailto:organizers-los-angeles@kubernetescommunitydays.org)
* [x] Region created in Bevy
* [x] Added to Bevy header
* [ ] Marketing Plan shared

### Organizer Planning Timeline
* [ ] Make a copy of [this timeline](https://docs.google.com/spreadsheets/d/1bvCiyyDut1seSnBE6pzVevcJkXLeWbxbncvhFsyY8PI/edit#gid=1736864170) based on whether your event is In-person, Hybrid, or Virtual and link your own spreadsheet here ().
* [ ] To assure you have a diverse lineup, you are required to send a copy of your program to kcd@cncf.io before you publish it.

### Post event checklist for Organizer:
* [x] Send CNCF event analytics
* [x] Schedule post mortem meeting
* [x] Send CNCF videos to be uploaded to YouTube
* [x] Link videos from YouTube to event page
* [ ] Post event survey sent out

### Post event checklist for CNCF:
* [x] Removed from Bevy header
* [ ] ~~CFP deactivated~~
* [x] ~~Upload videos to YouTube~~ link playlist from Youtube

## Notes

KCD Los Angeles is hosted as a colocated event within [Southern California Linux Expo](https://www.socallinuxexpo.org/scale/20x). As such, all registration, sponsorship, finance, etc. is handled by the host event.  It is the successor to three successful "ContainerDays" events in the same location, with some of the same organizers.  It happens in parallel to DevOpsDays LA.

SCALE also handles upload of videos, so we'd be linking a playlist rather than uploading.

KCD LA is a two-day event with workshops on the 23rd and a full day of talks on the 24th.

**Comment by AudMonte01** [comment]
NOTES FROM YESTERDAY'S MEETING

CNCF to help promote by:
* Plugging in the marketing committee next week
* Social amplification
* KubeWeekly (READY)

To-dos
* Josh to send email addressed of organizers to Audra - Audra to create email alias

Audience
* Weekdays of Scale (KCD days) attracts more of the people using K8s
* Main event attracts more hobbyists - the cloud native track

Next meeting set for December 5

**Comment by AudMonte01** [comment]
Meeting Notes
* 56 submissions for KCD program via Scale 20x. May leverage Cloud Native track submissions if needed
* Acceptances will go out mid January
* 1 or 2 tracks (there is venue space for 2) - TBD

**Comment by AudMonte01** [comment]
MEETING NOTES

Have 2 rooms dedicated to content
* Beginner oriented room
* An intermediate and advanced room

4 Workshops
* Won't have alternates - if they end up having to cancel one, that's fine since they usually have 2 workshops

Waitlist
* Checking with Dawn Foster if she presents another session
* Bulk of acceptances for SCALE already went out, but handful are still in their back pocket
* Free ticket, but travel expenses are another story

Diversity (6 out of 18 session are non-male presenters)
* Workshops are week on non-white representation
* Pasadena Convention Center is 100% accessible
* Cost for attendees is reasonable

COVID
* Sanitize in-between speakers
* Masks are recommended but not required 

Announce Schedule
* This upcoming weekend
* New reg goal is 120 minimum, but two rooms can fill up to 500

107 Submissions
* More than double the submissions this year from previous years, and many new submissions (widely successful)
* Quality of submissions went up as well
* Only 15% were product pitches

**Comment by AudMonte01** [comment]
3/21/2023 RETROSPECTIVE MEETING

What went well?
* Workshops were well attended
* Speakers had good feedback
* KCD talks were parallel, but different enough to make it easy for attendees to decided
* Side by side with DevOps Days worked really well
* Weren't able to pull together a social event for the KCD

What didn't go so well
* Google network issues that didn't get resolved until after the KCD talks were over
* Only 2 Sponsors came in last-minute (time and money)
-- next year they want to create a prospectus and do some things they weren't able to do before like a dedicated event
* in-person slides need to have larger text (remind speakers next year)
* Have Kube 101 workshop to prevent time wasted for proctors.
* More interaction with workshop presenters ahead of time to get audience prep

General notes
* Beginner-oriented talks had much better attendance, maybe 3x any “advanced” talks.

**Comment by jberkus** [comment]
This event is concluded.

Final report:

KCD Los Angeles Event Report
Josh Berkus

## Background

Southern California Linux Expo (SCALE) is a general, non-profit open source conference that has been held 20 times over a period of 22 years in the Los Angeles area, as covered in prior trip reports.  For the last five SCALE events, I've run a container-related track at SCALE as volunteer conference staff. This has previously been a single-room track that attracted around 120 peak attendees.  I run it with a small program committee.

Shortly before COVID, the CNCF established the Kubernetes Community Day program.  Under it, local organizers can run small-scale (or sometimes not-so-small) cloud native events officially supported by the CNCF.  Some of these have been online, but in-person ones have recently been held in Paris, Amsterdam, and Washington DC.  Red Hat sponsors individual KCDs, selected according to several criteria.

KCD-LA was on March 9 and 10.

## Launching KCD-LA 2023

This year, we reached out to the CNCF for their support in expanding the cloud native content at SCALE, and for permission to brand it as "Kubernetes Community Day Los Angeles".  Our main goal was to increase talk submissions from new, and new-to-LA speakers, as prior cloud native tracks had featured a lot of repeat speakers.  We also expanded the program committee from three to five (including folks from Red Hat, VMware, AWS, and Pulumi). We set up the first two days (Thursday and Friday) as KCD-LA, and left the Saturday-Sunday program as the SCALE cloud native track.  This took into account that the Saturday-Sunday schedule is not as focused, and let us take talks on the weekend that were not CNCF-related, but were still cloud.

Having KCD-LA hosted by SCALE made it unusual for a KCD, particularly since no direct funding was required.  As such, this year we chose not to fundraise, but to instead direct sponsors to the main SCALE conference. That will change next year. We also used SCALE's CfP system and registration, using the official KCD-LA page to provide extra information and redirect people to SCALE. One tremendous advantage this gave us was putting KCD-LA right next to DevOpsDays LA, which resulted in a lot of crossover of attendees. The main disadvantage is that we had no real way to know what our total attendance was, just headcounts for each session.

## Program Content

The speaker recruitment was extraordinarily successful, with 104 talk submissions and 16 workshop submissions.  On the basis of this response, we decided to expand the program from one two two rooms, and from two workshops to four.  Thursday's program was all workshops, and Friday was two rooms with seven sessions each. We arranged it so that one room was more "beginner, developer" content, and the other was more "deep dive cloud native." Unsurprisingly, the first room was considerably more full than the second.

Topics covered included Gateway API, secrets, GitOps, SigStore, kubebuilder, instrumentation, eBPF, WASM, AI/ML, and edge. The three most popular sessions were Kaslin and Christina's Kubernetes intro, RBAC 101, and Minimum Viable Kubernetes, showing the LA community's interest in introductory topics.  Due to a last-minute speaker cancellation, Jeefy, Justin, and Steve ended up having a "Kubernetes Home Lab" BOF at the end of the conference day, which turned into one of the more popular sessions.

Our program featured quite a few first-time speakers, including one who gave her first-ever workshop. KCDs provide a great opportunity for new presenters to reach a Kubernetes audience. Even better, both because of our direct efforts and because of SCALE's reputation, we were able to attract a relatively diverse speaker pool, including seven women speakers, and at least four people of color.

## Attendees

Since KCD-LA was embedded in a larger event, we can't ever know how many people attended at least one session.  Particularly, many folks were crossing over between DevOpsDays LA and the KCD.  The most we had in both rooms at the same time was 176 attendees, suggesting that something like 200 to 300 individuals attended some part of KCD-LA.

Unlike many other Cloud Native events, attendees at KCD-LA were generally close the start of the adoption path. Audiences included hobbyists, front-end developers, admins at non-tech industries, and many other people who were either just getting started with Kubernetes or had come to KCD-LA to begin learning about it. We did have some cloud native contributors and power users, but they were definitely a minority.  The audience was an estimated 80% local to Southern California.

## Plans for Next Year

Given the success of the event, we plan to repeat KCD-LA next year. This time, we will also look for sponsors in order to support a social event, travel scholarships, and other event features we can't do otherwise. We will also coordinate more closely with DevOpsDays LA, as many audience members and speakers will continue to regard the two as a single big event.  We will also look at choosing content that leans even more into the introductory direction, and whether we should change how the schedule is arranged.

## Thanks

I definitely need to thank the whole KCD Program Committee (Guinevere Saenger, Steve Wong, Justin Garrison, and Savitha Raghunathan), the CNCF staff both on and offsite, and the tremendous SCALE volunteer organization, for making KCD-LA possible.

**Comment by AudMonte01** [comment]
Videos are LIVE here: https://www.youtube.com/playlist?list=PLj6h78yzYM2MXW5NBSnSOqsuvi4QO2ICT

I am going to close this KCD issue now

**Comment by AudMonte01** [comment]
OCTOBER 10 MEETING NOTES

Sponsorships
* Are hurting due to KubeCon Paris, so might not be able to have a social or sponsor speaker travel

Content
* 22 submissions and 3 weeks out (not including DoKC)
* 26 potential speakers showed up to CFP writing workshop last Friday

Marketing
* Use KCD booth at KubeCon Chicago for curated content
* [ ] team to decided how they want to use 10 LF exam vouchers. Audra to send codes.
